libs/sdk-ui-dashboard/src/model/store/insights/insightsSelectors.ts

```diff
@@ -71,3 +92,145 @@ export const selectInsightByWidgetRef: (ref: ObjRef | undefined) => DashboardSel
             return ref && insights.get(ref);
         });
     });
+
+/**
+ * Selects raw export custom measure overrides for insight by ref.
+ *
+ * @privateRemarks
+ * For a given insight (identified by its ref), this selector will return a mapping of measure localId to
+ * the custom title that should be used when raw exporting the insight.
+ * The order of precedence is:
+ * 1. If the measure has an alias in the insight definition, use that.
+ * 2. If the measure has a title in the insight definition, use that.
+ * 3. Use the title of the measure from the catalog.
+ *
+ * @alpha
+ */
+const selectRawExportMeasureOverridesForInsightByRef: (
+    ref: ObjRef | undefined,
+) => DashboardSelector<IRawExportCustomOverrides["measures"] | undefined> = createMemoizedSelector(
+    (ref: ObjRef | undefined) => {
+        return createSelector(selectInsightByRef(ref), selectCatalogMeasures, (insight, catalogMeasures) => {
+            if (!insight) {
+                return undefined;
+            }
+
+            return insightMeasures(insight).reduce((overrides, measure) => {
+                const localId = measureLocalId(measure);
+
+                // first, try getting the title from the insight itself, giving precedence to the alias over the title
+                const titleFromInsightMeasure = measureAlias(measure) || measureTitle(measure);
```

Comment:
We probably don't allow empty string aliases, right? (asking because of || versus ??)